utils/kubernetes/kubeutil.py

```diff
@@ -459,23 +549,32 @@ def get_node_hosttags(self):
         # API server version
         try:
             request_url = "%s/version" % self.kubernetes_api_root_url
-            master_info = self.retrieve_json_auth(request_url)
+            master_info = self.retrieve_json_auth(request_url).json()
             version = master_info.get("gitVersion")
             tags.append("kube_master_version:%s" % version[1:])
         except Exception as e:
             # Intentional use of non-safe lookups to get the exception in the debug logs
             # if the parsing were to fail
             log.debug("Error getting Kube master version: %s" % str(e))
 
-        # Kubelet version
+        # Kubelet version & labels
+        if not self.init_success:
+            log.warning("Kubelet client failed to initialize, kubelet host tags will be missing for now.")
+            return tags
         try:
             _, node_name = self.get_node_info()
             if not node_name:
                 raise ValueError("node name missing or empty")
             request_url = "%s/nodes/%s" % (self.kubernetes_api_url, node_name)
-            node_info = self.retrieve_json_auth(request_url)
+            node_info = self.retrieve_json_auth(request_url).json()
             version = node_info.get("status").get("nodeInfo").get("kubeletVersion")
             tags.append("kubelet_version:%s" % version[1:])
+
+            node_labels = node_info.get('metadata', {}).get('labels', {})
+            for l_name, t_name in self.kube_node_labels.iteritems():
```

Comment:
is this our code? could you please rename l_name to label_name or just label.

Reply:
No this is not our code, so I don't think it is worth it to deviate from their code for this.